### PR TITLE
EDSL Fluent API: convolution default parameters fix

### PR DESCRIPTION
**Pull request by 1sanz**
Issue with initialization was observed while working on onnxRT

### CODE REVIEW

**Comment by flaub** [comment]
Interesting, what was the problem you were running into? I don't see how these changes would fix anything unless you were perhaps using an older compiler which doesn't properly handle C++11 initialization (or maybe this is a C++14 feature?). I've seen older compilers that also failed here, so perhaps we should try to make the client-side EDSL code as backwards compatible as we can (basically only use C++11 features).

File: plaidml/op/op.h [hunk 1]
```diff
@@ -248,15 +258,15 @@ class convolution {
   std::vector<int> filter_shape_;    // Default: empty (i.e. no filter dim check)
   int groups_ = 1;
```
Comment (flaub) [inline plaidml/op/op.h:259]: Possibly missed a spot here.

**Comment by tzerrell** [comment]
> Interesting, what was the problem you were running into?

Unset defaults manifesting as the enum validation code throwing `std::runtime_error("Invalid enumeration value")`. This went away if the default value was explicitly passed by calling the setter function.

File: plaidml/op/op.h (hunk 1 above)
Comment (1sanz) [reply plaidml/op/op.h]: fixed.

**Comment by 1sanz** [comment]
> LGTM except the overlooked `groups_` default nit @flaub mentioned.
> 
> (I take it this approach did ultimately resolve the issue?)

yes. At first I thought it was not helping but turned out that was a cmake cache issue.

**Pull request merged**